test/extended/cmd.sh

```diff
@@ -147,6 +147,12 @@ os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "extended/cmd/new-build"
+os::cmd::expect_success "oc new-build https://github.com/openshift/ruby-hello-world.git#bd94cbb228465d30d9d3430e80b503757a2a1d97"
+os::cmd::try_until_text "oc logs builds/ruby-hello-world-1" "bd94cbb228465d30d9d3430e80b503757a2a1d97"
```

Comment:
please add a comment so we know what this is actually supposed to be looking for.  also make the text it's looking for more specific to the exact line you've added ("commit: xyz").